apps/code-examples/src/app/code-examples/datetime/datepicker/basic/datepicker-demo.component.ts

```diff
@@ -9,13 +9,11 @@ import {
   selector: 'app-datepicker-demo',
   templateUrl: './datepicker-demo.component.html',
 })
-export class DatepickerDemoComponent implements OnInit {
+export class DatepickerDemoComponent {
   public myForm: UntypedFormGroup;
 
-  constructor(private formBuilder: UntypedFormBuilder) {}
-
-  public ngOnInit(): void {
-    this.myForm = this.formBuilder.group({
+  constructor(formBuilder: UntypedFormBuilder) {
+    formBuilder.group({
```

Comment:
This should assign `this.myForm`, right?

Reply:
Yes. That was an accidental change. Fixed.